service/src/users/users.service.ts

```diff
@@ -0,0 +1,28 @@
+import { Injectable } from '@nestjs/common';
+import { UserDto } from './dto/user.dto/user.dto';
+
+@Injectable()
+export class UsersService {
+    public users: UserDto[] = [];
+    
+    findAll():UserDto[]{
+        return this.users;
+    }
+
+    findOne(id?: number, email?: string):UserDto {
```

Comment:
I find it's better to avoid variety of methods as findById and findByEmail when there isn't complicated logic that would make generalised method complicated.